ports/esp32/usb.c

```diff
@@ -58,6 +59,7 @@ static void usb_callback_rx(int itf, cdcacm_event_t *event) {
                 ringbuf_put(&stdin_ringbuf, usb_rx_buf[i]);
             }
         }
+        xTaskNotifyGive(mp_main_task_handle);
```

Comment:
I think it would be good to put this in a function called `mp_hal_wake_main_task()`.

Reply:
done.